.changeset/friendly-swans-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/view-components": minor
```

Comment:
I'd consider this `minor` because we are changing the supported versions of Ruby/Rails.